src/main/java/uk/gov/hmcts/reform/divorce/orchestration/domain/model/exception/CaseAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.divorce.orchestration.domain.model.exception;
+
+public class CaseAlreadyExistsException extends Exception {
```

Comment:
Could this extend the BaseException, then GlobalExceptionHandler would not need any changes?